CHANGELOG.md

```diff
@@ -77,6 +77,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Bug Fixes
 
+* (baseapp) [#18609](https://github.com/cosmos/cosmos-sdk/issues/18609) Gas Consumption idiosyncracies in BeginBlock / DeliverTx
```

Comment:
This is a major fix, I'd say this warrants a bit more description.

Something like:

> Fixed idiosyncrasies in the block gas meter after BeginBlock and before DeliverTx, ensuring transaction processing always starts with the expected block gas meter.

Reply:
ACK, updating in new commits.